sdk/fmt.sh

```diff
@@ -49,7 +49,7 @@ check_diff() {
   # $1 merge_base
   # $2 regex
   # "${@:3}" command
-  changed_files=$(git diff --name-only --diff-filter=ACMRT "$1" | grep $2 | grep -E -v '^canton(-3x)?/' || [[ $? == 1 ]])
+  changed_files=$(git diff --name-only --diff-filter=ACMRT "$1" | grep '^sdk/' | sed 's|^sdk/||' | grep $2 | grep -E -v '^canton(-3x)?/' || [[ $? == 1 ]])
```

Comment:
is `grep '^sdk' even needed?

Reply:
I guess not - I was thinking of shoving the filter into the sed command anyways. However, if we start ever having .hs files outside that directory I'd rather that this script didn't fail in mysterious ways.